contributing/versions.md

```diff
@@ -13,23 +13,27 @@ MDC's version numbers strictly follow [semantic versioning](http://semver.org/):
 
 Major releases can contain breaking changes to clients, while minor and patch releases cannot.
 
-## What are breaking changes?
+## How do we determine which numbers need to change?
 
-- API deletions or modifications.
+We follow the guidelines defined by the [semver v2.0.0](https://semver.org/spec/v2.0.0.html#semantic-versioning-specification-semver) specification.
+
+## Roughly-speaking, what are breaking changes?
+
+- Build-breaking modifications for either Objective-C or Swift code.
 - Visible changes to user interface components.
 
-## What are non-breaking changes?
+## Roughly-speaking, what are non-breaking changes?
 
 - API additions.
 - Behavioral changes.
```

Comment:
What is a "behavioral change" if not visible? Is this where accessibility comes in?

Reply:
Accessibility fits that category somewhat, but the more I think about it the more I think we should likely remove these deviations from the semver definitions and focus strictly on breaking API changes.